topic-operator/src/main/java/io/strimzi/operator/topic/TopicOperatorUtil.java

```diff
@@ -270,4 +229,59 @@ public static String configValueAsString(String key, Object value) {
         }
         return valueStr;
     }
+
+    /**
+     * Create Cruise Control client.
+     *
+     * @param config Topic Operator configuration.
+     * @return Cruise Control client.
+     */
+    public static CruiseControlClient createCruiseControlClient(TopicOperatorConfig config) {
+        var sslCertificate = config.cruiseControlSslEnabled() ? 
+            TopicOperatorUtil.getFileContent(config.cruiseControlCrtFilePath()) : null;
+        var apiUsername = config.cruiseControlAuthEnabled() ? 
+            new String(TopicOperatorUtil.getFileContent(config.cruiseControlApiUserPath()), StandardCharsets.UTF_8) : null;
+        var apiPassword = config.cruiseControlAuthEnabled() ? 
+            new String(TopicOperatorUtil.getFileContent(config.cruiseControlApiPassPath()), StandardCharsets.UTF_8) : null;
+
+        if (config.cruiseControlHostname() == null || config.cruiseControlHostname().isBlank()) {
+            throw new IllegalArgumentException("Cruise Control hostname is not set");
+        }
+        if (config.cruiseControlSslEnabled() && (sslCertificate == null || sslCertificate.length == 0)) {
+            throw new IllegalArgumentException("Cruise Control certificate is not set");
+        }
+        if (config.cruiseControlAuthEnabled()) {
+            if (apiUsername == null || apiUsername.isBlank()) {
+                throw new IllegalArgumentException("Cruise Control username is not set");
+            }
+            if (apiPassword == null || apiPassword.isBlank()) {
+                throw new IllegalArgumentException("Cruise Control password is not set");
+            }
+        }
+
+        return CruiseControlClient.create(
+            config.cruiseControlHostname(),
+            config.cruiseControlPort(),
+            config.cruiseControlRackEnabled(),
+            config.cruiseControlSslEnabled(),
+            sslCertificate,
+            config.cruiseControlAuthEnabled(),
+            apiUsername,
+            apiPassword
+        );
+    }
+
+    /**
+     * Get file content.
+     * 
+     * @param filePath File path.
+     * @return file content as bytes.
+     */
+    public static byte[] getFileContent(String filePath) {
```

Comment:
I think it cannot be private ... but it can be probably `/* test */`?

```suggestion
    /* test */ static byte[] getFileContent(String filePath) {
```

Or did I missed something somewhere?